tst/EnergyPlus/unit/FaultsManager.unit.cc

```diff
@@ -192,3 +200,66 @@ TEST_F( EnergyPlusFixture, FaultsManager_FaultFoulingAirFilters_CalFaultyFanAirF
 	Fan.deallocate();
 
 }
+
+TEST_F( EnergyPlusFixture, FaultsManager_FaultChillerSWTSensor_CalFaultChillerSWT )
```

Comment:
@rongpengzhang These are good tests for CalFaultChillerSWT, but there is a lot of new code sprinkled among the various chiller models.  The new integration test covers only one chiller type Chilled:Electric:Reformulated:EIR.  Do you have integration test files for all of the other chiller types?  I am not suggesting that we add that many new test suite files, but we need to test all of this new code with something.